.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Rust
+
+on:
+  push:
+  pull_request:
+
+env:
+  WINFSP_VERSION: "2.0.23075"
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: windows-latest
```

Comment:
I recommend using a specific version of windows runner, not the latest to limit possible breaking change